src/bootstrap/builder.rs

```diff
@@ -1111,6 +1111,17 @@ impl<'a> Builder<'a> {
             );
         }
 
+        // If Control Flow Guard is enabled, pass the `control_flow_guard=checks` flag to rustc
+        // when compiling the standard library, since this might be linked into the final outputs
+        // produced by rustc. Since this mitigation is only available on Windows, only enable it
+        // for the standard library in case the compiler is run on a non-Windows platform.
```

Comment:
Seems like the logic for doing this only on Windows was lost? It would also be good to mention this in config.toml.example.

Reply:
Thanks! I've added the Windows check and improved the explanation in config.toml.example.